projects/core/src/occ/adapters/product/converters/occ-product-search-page-normalizer.ts

```diff
@@ -20,11 +20,40 @@ export class OccProductSearchPageNormalizer
       ...target,
       ...(source as any),
     };
+    this.normalizeUselessFacets(target);
     if (source.products) {
       target.products = source.products.map(product =>
         this.converterService.convert(product, PRODUCT_NORMALIZER)
       );
     }
     return target;
   }
+
+  /**
+   * The (current) backend returns facets with values that do not contribute
+   * to the facet navigation much, as the number in the result list will not get
+   * affected when they got applied. A ticket has been created to fix this
+   * behaviour, see https://jira.hybris.com/browse/CS-427.
```

Comment:
Is there an issue for this removal on GitHub? Maybe we can just add `TODO:` with the ticket number, so that we don't forget to cleanup this?

Reply:
it feels like it's lightyears ahead of us, I can life without the ticket for now.